tests/garage/envs/pybullet/test_pybullet_env.py

```diff
@@ -0,0 +1,106 @@
+"""Test module for PybulletEnv"""
+import pickle
+
+from pybullet_envs.bullet.cartpole_bullet import CartPoleBulletEnv
+from pybullet_envs.bullet.cartpole_bullet import CartPoleContinuousBulletEnv
+from pybullet_envs.bullet.kukaCamGymEnv import KukaCamGymEnv
+from pybullet_envs.bullet.kukaGymEnv import KukaGymEnv
+from pybullet_envs.bullet.minitaur_duck_gym_env import MinitaurBulletDuckEnv
+from pybullet_envs.bullet.minitaur_gym_env import MinitaurBulletEnv
+from pybullet_envs.bullet.racecarGymEnv import RacecarGymEnv
+from pybullet_envs.bullet.racecarZEDGymEnv import RacecarZEDGymEnv
+from pybullet_utils.bullet_client import BulletClient
+import pytest
+
+from garage.envs.pybullet import PybulletEnv
+from tests.helpers import step_env
+
+
+@pytest.fixture
+def envs():
```

Comment:
i think you want a parameterized test here, not a fixture. Fixtures are generally for shared setup, rather than enumerating tests.